spec/factories/allocation.rb

```diff
@@ -8,6 +8,7 @@
 
     prisoner_category { Allocation.prisoner_categories.values.sample }
     sentence_length { Allocation.sentence_lengths.values.sample }
+    estate { Allocation.estates.values.sample }
```

Comment:
do you want to allow for a nil value too?

Reply:
The estate can be overridden if we want some different behaviour. These attributes all support nil values so I suppose there's no real need to set them in the factory 🤔